dev.hdos.HDOS.json

```diff
@@ -0,0 +1,90 @@
+{
+    "id": "dev.hdos.HDOS",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "hdos",
+    "separate-locales": false,
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk11"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--env=JAVA_HOME=/app/jre",
+        "--talk-name=org.freedesktop.Notifications",
+        "--filesystem=xdg-run/app/com.discordapp.Discord:create",
+        "--filesystem=~/hdos:create",
+        "--filesystem=~/jagex_runescape_preferences.dat:create",
```

Comment:
`create` creates directories, not empty files

Reply:
Yeah, was wondering that, I'll fix it when I'm home